methods.py

```diff
@@ -777,7 +777,8 @@ def __getitem__(self, k: str):
         if env.get("module_mono_enabled"):
             import modules.mono.build_scripts.mono_reg_utils as mono_reg
 
-            mono_root = env.get("mono_prefix") or mono_reg.find_mono_root_dir(env["bits"])
+            bits = "32" if env["arch"] in ["arm32", "x86_32"] else "64"
```

Comment:
Another option is `bits = "32" if "32" in env["arch"] else "64"`, it may seem more hacky but it should always work since it would be weird for an architecture to contain `32` without it being 32-bit.